bin/script-util.js

```diff
@@ -83,6 +91,27 @@ function isTypeScriptPackage(pkg) {
   return fs.existsSync(path.join(pkg.location, 'tsconfig.json'));
 }
 
+/**
+ * Run an async function
```

Comment:
Please improve the comment to describe what is going to happen and under which conditions.